### ISSUE TITLE
xIisModule: Write-Verbose runtime errors

### ISSUE THREAD

**Issue by vercellone** [PowerShell/xWebAdministration#215]
### A parameter cannot be found that matches parameter name 'f'..

``` PowerShell
    # Line 279 of MSFT_xIisModule.psm1 v1.13.0.0 
    # needs parenthesis
    Write-Verbose -Message $LocalizedData.VerboseGetIisHandler -f $Name,$SiteName
```
### A positional parameter cannot be found that accepts argument 'S'.

``` PowerShell
    # Line 389 of MSFT_xIisModule.psm1 v1.13.0.0 
    # S needs to be removed
    Write-Verbose -Message ($LocalizedData.VerboseTestTargetResourceImplModulePresent `
                            -f $ModulePresent)S
```

**Comment by mbreakey3** [comment]
Working on cleaning up this module - should be fixed in PR #219

**Comment by rosberglinhares** [comment]
This issue is fixed in v1.17.0.0. Shouldn't it be closed?
